OlesiaBarannik/Cat/cat.py

```diff
@@ -0,0 +1,98 @@
+from tkinter import *
```

Comment:
good job

look at this version of code

from tkinter import Tk, Label, Button, PhotoImage, messagebox as mb
from tkinter import ttk

class CatGame:
    def __init__(self):
        self.root = Tk()
        self.root.title("CAT")

        icon = PhotoImage(file="iconCat.png")
        self.root.iconphoto(False, icon)

        self.healthCat = 20
        self.leisureCat = 20
        self.game = 3

        self.create_widgets()
        self.update_clock()

        self.root.mainloop()

    def create_widgets(self):
        label2 = Label(width=27, height=3, text="It is your Cat", font="Arial")
        b1 = ttk.Button(width=15, text="feed", command=self.health)
        b2 = ttk.Button(width=15, text="caress", command=self.leisure)
        b3 = ttk.Button(width=15, text="train", command=self.health)
        b4 = ttk.Button(width=15, text="play", command=self.leisure)
        b5 = ttk.Button(width=15, text="Exit", command=self.root.destroy)
        l1 = Label(width=20, height=3, text="health - " + str(self.healthCat) + "%")
        l2 = Label(width=20, height=2, text="leisure - " + str(self.leisureCat) + "%")
        l3 = Label(width=20, height=2, text="your cat is alive")

        label2.grid(row=0, column=2, columnspan=3, rowspan=2)
        b1.grid(row=2, column=0)
        b2.grid(row=3, column=0)
        b3.grid(row=4, column=0)
        b4.grid(row=5, column=0)
        b5.grid(row=9, column=3)
        l1.grid(row=6, column=0)
        l2.grid(row=7, column=0)
        l3.grid(row=7, column=3)

        self.logo = PhotoImage(file="cat.png")
        logo1 = Label(image=self.logo)
        logo1.grid(row=2, column=2, columnspan=5, rowspan=5)

        self.labels = {
            'health': l1,
            'leisure': l2,
            'status': l3
        }

    def update_clock(self):
        now = time.strftime("%H:%M:%S")
        self.root.after(1000, self.update_clock)
        self.heppimin()

    def heppimin(self):
        if self.healthCat <= 0 or self.leisureCat <= 0:
            answer = mb.askyesno(title="You lost", message="Do you want to play again?")
            if answer:
                self.healthCat += 50
                self.leisureCat += 50
            else:
                 self.game = 1
        elif self.healthCat >= 100 and self.leisureCat >= 100:
            answer2 = mb.askyesno(title="You win", message="Do you want to play again?")
            if answer2:
                self.healthCat -= 50
                self.leisureCat -= 50
            else:
                 self.game = self.game + 1
        self.healthCat -= 3
        self.leisureCat -= 3
        self.labels['health'].configure(text="health - " + str(self.healthCat) + "%")
        self.labels['leisure'].configure(text="leisure - " + str(self.leisureCat) + "%")

    def health(self):
        self.healthCat += 10
        self.leisureCat -= 2
        self.labels['health'].configure(text="health - " + str(self.healthCat) + "%")
        self.labels['leisure'].configure(text="leisure - " + str(self.leisureCat) + "%")
        self.labels['status'].configure(text="your cat is healh")
        if self.leisureCat <= 10:
            mb.showerror("sorry", "your cat is sad")
            self.leisureCat += 20

    def leisure(self):
        self.healthCat -= 2
        self.leisureCat += 10
        self.labels['health'].configure(text="health - " + str(self.healthCat) + "%")
        self.labels['leisure'].configure(text="leisure - " + str(self.leisureCat) + "%")
        self.labels['status'].configure(text="your cat is leisure")
        if self.healthCat <= 10:
            mb.showerror("sorry", "your cat is ill")
            self.healthCat += 20

CatGame()